index.html

```diff
@@ -0,0 +1,293 @@
+              <img src="assets/icons/circle-icon.svg" alt="circlular icon" />
+              <span class="subheading-list-title">2015</span>
+            </div>
+            <section class="card-content">
+              <p>
+                A daily selection of privately personalized reads; no accounts
+                or sign-ups required.
+              </p>
+            </section>
+            <ul class="webdev-languages">
+              <li>HTML</li>
+              <li>CSS</li>
+              <li>JavaScript</li>
+            </ul>
+            <button class="see-project" type="button">See project</button>
+          </div>
+        </div>
+      </section>
+      <section class="about-myself">
+        <div>
+          <h2 class="greeting">About Myself</h2>
+          <div class="job-description">
+            <p>
+              I'm a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don't
+              hesitate to contact me.
+            </p>
+          </div>
+          <h2 class="connect-link">LET'S CONNECT</h2>
+          <div class="social-media-icons">
+            <ul class="social-media-links">
+              <li>
+                <a href="#"
+                  ><img src="assets/images/twitter-logo.png" alt="twitter-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="assets/images/Linkedin icon.png"
+                    alt="linkedin-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img src="assets/images/medium-icon.png" alt="medium-icon"
+                /></a>
+              </li>
+              <li>
+                <a href="#">
+                  <img src="assets/images/github-logo.png" alt="github-logo"
+                /></a>
+              </li>
+              <li>
+                <a href="#"
+                  ><img
+                    src="assets/images/angel-list-logo.png"
+                    alt="angel-list-logo"
+                /></a>
+              </li>
+            </ul>
+          </div>
+          <button class="my-resume" type="button">Get my resume</button>
+        </div>
+        <ul id="aboutme-list">
+          <li class="list-title">
+            <ul class="lists">
+              <li><h4>Languages</h4></li>
+              <li>
+                <img
+                  src="assets/icons/dropdown-icon.png"
+                  alt="dropdown arrow"
+                />
+              </li>
+            </ul>
+          </li>
+          <li class="programming-lang">
+            <ul>
+              <li>
+                <img
+                  src="assets/icons/js-icon.svg"
+                  alt="javascript icon"
+                />Javascript
+              </li>
```

Comment:
Great work done by applying most of changes requested by the previous reviewer but we still have to fix one last thing that appeared after changes , the items in this bloc are not correctly aligned I suggest that you remove the default left padding of the ul tag 

**yours**
![UL padding left](https://user-images.githubusercontent.com/59707859/235315516-d48e5617-c571-4342-a162-cc5dc3eed564.PNG)


**figma **
![ABOUT MY SELF REVIEW](https://user-images.githubusercontent.com/59707859/235315450-0b017d57-95cb-47f1-93fc-653950a1f466.png)